ansible/files/admin_api_scripts/pg_upgrade_scripts/initiate.sh

```diff
@@ -77,6 +77,10 @@ cleanup() {
         fi
     fi
 
+    if [ -d "/usr/share/postgresql/${PGVERSION}.bak" ]; then
```

Comment:
What's responsible for creating this? A failed attempt?

Reply:
Yep. It's also part of the cleanup job after upgrade completion in case that was ever backed up.

The change also seems to have been included already in a previous PR - will close this one